tests/phpunit/tests/KSES.php

```diff
@@ -308,6 +332,25 @@ public function test_filter_kses_allowed_html( $html, $expected ) {
 		remove_filter( 'wp_kses_allowed_html', [ $kses, 'filter_kses_allowed_html' ] );
 	}
 
+	/**
+	 * Testing the filter_kses_allowed_html() method.
+	 *
+	 * @covers ::filter_kses_allowed_html
+	 * @covers ::array_merge_recursive_distinct
+	 */
+	public function test_filter_kses_allowed_html_filter() {
```

Comment:
```suggestion
	public function test_filter_kses_allowed_html_uses_deep_merge() {
```

Otherwise the name is way too similar to the other test